client/verta/verta/_deployment/endpoint.py

```diff
@@ -396,6 +396,29 @@ def get_access_token(self):
             return None
         return tokens[0]['creator_request']['value']
 
+    def create_access_token(self, token):
+        """
+        Creates an access token for the Endpoint.
+
+        Parameters
+        ----------
+        token : str
+            Token to create.
+
+        """
+        if not isinstance(token, six.string_types):
+            raise TypeError("\"token\" must be a string.")
```

Comment:
Convention around this parts is to use backticks for parameter names
```suggestion
            raise TypeError("`token` must be a string.")
```